### DISCUSSION TITLE
TypeScript definitions or JSDoc ones for CJS library

### DISCUSSION THREAD

**Discussion by EmilyGraceSeville7cf** [linuxmint/discussions#601] [Questions] [answered]
Hi! I wanna write spices for Cinnamon, but where do I get CJS [TypeScript](https://github.com/DefinitelyTyped/DefinitelyTyped) definitions or JSDoc ones for CJS library from?

**Comment by anaximeno** [comment, marked as answer]
Check this https://github.com/linuxmint/cinnamon-spices-applets/tree/master/.typescript-declarations and see if it helps you.

**Reply by EmilyGraceSeville7cf** [reply]
Thanks, it's a really helpful answer 🙏
